src/ceph-daemon/ceph-daemon

```diff
@@ -1604,9 +1606,11 @@ def command_adopt():
         data_dir_src = ('/var/lib/ceph/%s/%s-%s' %
                         (daemon_type, args.cluster, daemon_id))
         data_dir_src = os.path.abspath(args.legacy_dir + data_dir_src)
-        data_dir_dst = make_data_dir(fsid, daemon_type, daemon_id)
-        for data_file in iglob(os.path.join(data_dir_src, '*')):
-            move_file(data_file, data_dir_dst, uid=uid, gid=gid)
+        data_dir_dst = make_data_dir(fsid, daemon_type, daemon_id,
+                                     uid=uid, gid=gid)
+        move_files(glob(os.path.join(data_dir_src, '*')),
```

Comment:
Ok, but this looks like we actually want a `move_dir()` here?

Reply:
yes, but the `data_dir_dst` already exists so that would create a `data_dir_src` as a subdir under `data_dir_dst` without chown'ing the files in the moved subdir.

The idea here was to treat this similar to this from the shell:
```
# mv src/* dst/
```